### ISSUE TITLE
[Suggestion] Config to allow stack size number UI to appear even on < 64 stack items

### ISSUE THREAD

**Issue by MrKashew** [frank89722/Staaaaaaaaaaaack#9]
Not sure if this is simple/worthwhile to implement given the scope of the mod may focus on only stacks > 64

**Comment by frank89722** [comment]
No, this mod made to combines drops without changing the vanilla stack mechanic.
Change how stack work usually break a lot of stuff

**Comment by frank89722** [comment]
Oh I miss understood what you're talking about lol
